cmds/core/kexec/kexec_linux.go

```diff
@@ -155,25 +230,27 @@ func main() {
 			if len(opts.dtb) > 0 {
 				dtb, err = os.Open(opts.dtb)
 				if err != nil {
-					log.Fatalf("Failed to open dtb file %s: %v", opts.dtb, err)
+					return fmt.Errorf("failed to open dtb file %s: %v", opts.dtb, err)
```

Comment:
should calls like this just always wrap the error for those who might come later?

Reply:
:+1: